docs/tool_jcmd.md

```diff
@@ -29,15 +29,24 @@ Use the `jcmd` tool to run diagnostic commands on a specified VM.
 
 The command syntax is as follows:
 
-    jcmd [<options>] [<vmid> <arguments>]
+    jcmd [<options>] [<vmid | display name | 0> <arguments>]
 
 Where:
 
-- The available `<options>` are:    
+- The available `<options>` are:
+    - `-l`: lists current Java&trade; processes recognized by the `jcmd` tool. The list includes VMID, which is usually the *process ID* (pid) and the display name, which refers to the target Java VM process that can be attached by `jcmd`. `-l` is the default option.
     - `-J`: supplies arguments to the Java VM that is running the `jcmd` command. You can use multiple `-J` options, for example: `jcmd -J-Xmx10m -J-Dcom.ibm.tools.attach.enable=yes`
     - `-h`: prints the `jcmd` help
 
-- `<vmid>` is the Attach API virtual machine identifier for the Java&trade; VM process. This ID is often, but not always, the same as the operating system *process ID*. One example where the ID might be different is if you specified the system property `-Dcom.ibm.tools.attach.id` when you started the process. You can use the [`jps`](tool_jps.md) command to find the VMID.
+- `<vmid>` is the Attach API virtual machine identifier for the Java VM process. This ID is often, but not always, the same as the operating system pid. One example where the ID might be different is if you specified the system property `-Dcom.ibm.tools.attach.id` when you started the process. You can use the [`jps`](tool_jps.md) command to find the VMID.
+
+    You can also specify the full or partial target Java process display name instead of the VMID. The `jcmd` tool finds the corresponding VMID of the display name and runs the `jcmd` command.
+
+    You can specify the display name for a target VM through the [`com.ibm.tools.attach.displayName`](dcomibmtoolsattachdisplayname.md) system property. If the display name is not set through the system property, then the main class name along with the application arguments is set as the default display name. The command syntax is as follows:
+
+        jcmd [pid | main-class] command
```

Comment:
I still don't understand the purpose of this. The jcmd command syntax is already shown on line 32 as `jcmd <vmid | display name | 0> <arguments>`, why do we want to state it again here, in a paragraph describing the display name, and using different parameter names, ie. pid vs vmid, main-class vs display name, command vs arguments.